src/ChangeSet.ts

```diff
@@ -62,17 +62,29 @@ export class ChangeSet {
    * and end position. This is useful for showing the changes as groups in the UI.
    */
   get changeTree() {
-    const rootNodes: TrackedChange[] = []
+    const rootNodes: TrackedChange[][] = []
```

Comment:
same comment as in article editor about covering the array inside the type

Reply:
updated types to:
```
export type RootChanges = TrackedChange[][]
export type RootChange = TrackedChange[]
```